dfsclient/src/main/java/org/hpccsystems/dfs/client/HPCCRemoteFileWriter.java

```diff
@@ -156,7 +156,7 @@ public HPCCRemoteFileWriter(FileWriteContext ctx, DataPartition dp, IRecordAcces
 
         Attributes attributes = Attributes.of(  AttributeKey.stringKey("server.0.address"), primaryIP,
                                                 AttributeKey.stringKey("server.1.address"), secondaryIP,
-                                                ServerAttributes.SERVER_PORT, Long.valueOf(dp.getPort()));
+                                                AttributeKey.stringKey("server.port"), "" + dp.getPort());
```

Comment:
unless this approach performs better, we should use explicit functionality like String.valueOf()